src/multibody/fcl.hpp

```diff
@@ -189,23 +152,27 @@ struct GeometryObject
   /// \brief Name of the geometry object
   std::string name;
 
+  /// \brief Index of the parent frame
+  FrameIndex parentFrame;
```

Comment:
Fair enough. After reviewing the code entirely, it seems that the second term of the assert is not mandatory, although it is always true this only your parser is creating this. It would be nice to doublecheck if this assertion is needed or not, and add it in the doc if it is.